### ISSUE TITLE
A problem with test.py on MOT17

### ISSUE THREAD

**Issue by fixtern** [xingyizhou/CenterTrack#77]
Hi every one, 
Many thanks to the authors that let others have a chance to study over their work. I encountered a problem once I ran test.py. It looks like  convert_mot_to_coco.py put some folders on the test stage. For example after running we may see MOT17-02-DPM on test vector which is coming from the training folder. That's why we can't run the test.py conveniently. please take a look at this problem and fix it on the repo.

**Comment by xingyizhou** [comment]
I didn't understand the question. What happened exactly? How can I reproduce the issue (i.e., your command and data folder structures)? I can run the testing on my side following the instructions.

**Comment by fixtern** [comment]
Actually the problem goes back to convert_mot_to_coco.py, line 17. If we set HALF_VIDEO = True, then no matter the stage is train or test, it assigns train directory to the data_path which is not supposed to be. 
line 10: SPLITS = ['train_half', 'val_half', 'train', 'test']
line17: data_path = DATA_PATH + (split if not HALF_VIDEO else 'train')

**Comment by xingyizhou** [comment]
This is expected, since the validation set we used is from the training set.

**Comment by fixtern** [comment]
ok, thanks. So you mean if we want to perform the actual test on the test folder of MOT, this script is not designed for this aim, am I right?

**Comment by xingyizhou** [comment]
Correct.
